python/CMakeLists.txt

```
@@ -561,6 +572,19 @@ if(PYARROW_BUILD_CUDA)
   set_source_files_properties(pyarrow/_cuda.pyx PROPERTIES CYTHON_API TRUE)
 endif()
 
+# Acero
+if(PYARROW_BUILD_ACERO)
+  if(PYARROW_BUNDLE_ARROW_CPP)
+    bundle_arrow_lib(${ARROW_ACERO_SHARED_LIB} SO_VERSION ${ARROW_SO_VERSION})
+    if(MSVC)
+      bundle_arrow_import_lib(${ARROW_ACERO_IMPORT_LIB})
+    endif()
+  endif()
+
+  set(ACERO_LINK_LIBS ArrowAcero::arrow_acero_shared)
+  list(APPEND CYTHON_EXTENSIONS _acero _exec_plan)
```

```suggestion
  list(APPEND CYTHON_EXTENSIONS _acero)
```

(I just merged a PR that will need the above change)